### ISSUE TITLE
NullPointerException thrown by ScriptEngine eval method

### ISSUE THREAD

**Issue by nsenave** [InseeFr/Trevas#154]
When evaluating a VTL expression that a contains an operator not yet implemented, a `NullPointerException` is thrown.

A dedicated exception might be useful to identify this case more easily.

Example (version 0.3.0) :

```java
//
Dataset fooDataset = new InMemoryDataset(
        List.of(List.of(1L, 31L)),
        List.of(new Structured.Component("ID", Long.class, Dataset.Role.IDENTIFIER),
                new Structured.Component("FOO", Long.class, Dataset.Role.MEASURE))
);
//
SimpleBindings bindings = new SimpleBindings();
bindings.put("ds", fooDataset);
//
ScriptEngine engine = new ScriptEngineManager().getEngineByName("vtl");
engine.setBindings(bindings, ScriptContext.ENGINE_SCOPE);
engine.eval("ds := ds [ calc FOO_SUM := sum(FOO)];");
```

**Comment by FranckCo** [comment]
Example VTL script is not valid, but the thrown exception should indeed be more specific.
Valid VTL: `res := ds1[aggr a := sum(name) group by country];`
which is supported.

**Comment by NicoLaval** [comment]
@nsenave group by is mandatory when sum is used as an `aggregate operator`, not in the `analysis operator` usage.

`nvl` is not implemented in 0.3.0, it will be in 0.4.0

**Comment by NicoLaval** [comment]
v1.0.0
